pyproject.toml

```diff
@@ -24,7 +24,7 @@ dependencies = [
     "python-dateutil",
     "requests",
     "setuptools",
-    "sympy>=1.8",
+    "sympy==1.11",
```

Comment:
I fix the version by now. It seems that still there are some places where comparisons against 0 fails. I plan to allow versions larger than 1.8 again once the other cases get fixed.

Reply:
For the next version, how do you feel about requiring Sympy 1.11 and greater? 

The next release drops support for Python 3.6 and after that I suppose we will drop support for Python 3.7.

Reply:
>
> The next release drops support for Python 3.6 and after that I suppose we will drop support for Python 3.7.

Change this. I see that the [`mathics-scanner/pyproject.toml` now requires Python 3.8 or later](https://github.com/Mathics3/mathics-scanner/blob/master/pyproject.toml#L18).  And this is done because Sympy 1.11 needs 3.8 or greater. 

Given this, we should change [pyproject.toml here](https://github.com/Mathics3/mathics-core/blob/master/pyproject.toml#L29) and drop Python 3.7 in the next release too. (We will have to adjust release testing)

And all of this tends to suggest not trying to support for Sympy 1.8. 

Your thoughts?